packages/flutter/test/painting/text_painter_test.dart

```diff
@@ -145,17 +145,73 @@ void main() {
       caretOffset = painter.getOffsetForCaret(ui.TextPosition(offset: text.length), ui.Rect.zero);
       expect(caretOffset.dx, painter.width);
 
-      // Test with trailing full-width space
-      const String textWithFullWidthSpace = 'A\u{3000}';
-      checkCaretOffsetsLtr(textWithFullWidthSpace);
-      painter.text = const TextSpan(text: textWithFullWidthSpace);
+      text = 'A';
```

Comment:
These 3 lines are no longer needed it seems?

Reply:
Ah I see you needed `A`'s horizontal advance. It's `14.0`.

Reply:
Yes. I checked the value at runtime and found it to be 14.0, but since it depends on the style of the painter, I calculated it.